### PR TITLE
android sdk 2.2.1 rns added

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ android sdk 2.2.1 rns added
* Jira/GitHub Issue numbers (if any): \ FFM-11658
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -15,7 +15,16 @@ Review the notes below for details about recent changes to Harness Feature Flags
...
+#### Last updated: June 14, 2024
+
+## June 2024
+
+### Android SDK
+
+#### Version 2.2.1
+
+**New features and enhancements**:
+ - Adds a new method for closing the SDK. The `closeWithFuture()` method can be used if you need to ensure the SDK has been closed before continuing e.g. re-initialising the SDK. (FFM-11658)
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:27]:
```suggestion
 - Added a new method for closing the SDK. The `closeWithFuture()` method can be used to ensure the SDK has been closed before continuing, for example, re-initializing the SDK. (FFM-11658)
```

Comment (brian-f-harness) [reply release-notes/feature-flags.md:27]: The FFM-11658 ticket is for your release notes work. Is there a Jira associated with the new feature you can use here for reference instead?

**Review by brian-f-harness — APPROVED** [review]
Left a couple minor suggestions and a question. LGTM otherwise.

File: release-notes/feature-flags.md (hunk 1 above)
Comment (jenniferopal) [reply release-notes/feature-flags.md:27]: yeah, there was. I typed the wrong jira number. my mistake.

**Comment by jenniferopal** [comment]
Reviewed and approved. Changes made. Merging.

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -15,7 +15,16 @@ Review the notes below for details about recent changes to Harness Feature Flags
...
+#### Last updated: June 14, 2024
+
+## June 2024
+
+### Android SDK
+
+#### Version 2.2.1
+
+**New features and enhancements**:
+ - Added a new method for closing the SDK. The `closeWithFuture()` method can be used to ensure the SDK has been closed before continuing, for example, re-initializing the SDK. (FFM-11625)
```
Comment (brian-f-harness) [inline release-notes/feature-flags.md:27]: @jenniferopal There's a Zendesk ticket we should add for reference.
[64818](https://harnesssupport.zendesk.com/agent/tickets/64818)

Comment (jenniferopal) [reply release-notes/feature-flags.md:27]: HI @brian-f-harness , I missed that. I'll add the ticket to my next PR where I'm adding the release notes for Flutter SDK.

**Pull request merged**